dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -333,4 +360,4 @@
         <Object ocdref="com.ibm.ws.http.log.access"/>
     </Designate>
 
-</metatype:MetaData>
+</metatype:MetaData>
```

Comment:
Can you just add the new line in at the end here?

Reply:
Done